dll/win32/msgina/lang/ja-JP.rc

```diff
@@ -177,39 +177,39 @@ BEGIN
     DEFPUSHBUTTON "OK", IDOK, 115, 179, 50, 14
 END
 
-IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 122
+IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 121
```

Comment:
```suggestion
IDD_SHUTDOWN_FANCY DIALOGEX 0, 0, 208, 122
```
as the shutdown dialogs in the other languages?

Reply:
@HBelusca I got this value from `msgina.dll` from Japanese version of Windows XP (albeit with several my own slight adjustments)
And it is since to make it to have similar width of languages which uses smaller fonts in order to not look weird.
![image](https://user-images.githubusercontent.com/35190743/149786368-d799bd09-af98-424e-b877-92afa3d2350f.png)

Reply:
@arnavbhatt288 And if you look at the msgina of e.g. english version, do they use yet another size?
Also, can you check what the size of the drawn pictures is, on this japanese version, vs. the english one?

Reply:
@HBelusca One unit larger in width.
![image](https://user-images.githubusercontent.com/35190743/149897303-d2bfd2af-4235-4127-a8b0-fc568cfac764.png)